macaroon/Classes/Views/Configuration/Styling/Customization/Drawable.swift

```diff
@@ -3,6 +3,20 @@
 import Foundation
 import UIKit
 
+public protocol BorderDrawable: UIView {}
+
+extension BorderDrawable {
+    public func drawBorder(_ border: Border) {
+        layer.borderColor = border.color.cgColor
+        layer.borderWidth = border.width
+    }
+    
+    public func removeBorder() {
+        layer.borderWidth = 0.0
+        layer.borderColor = nil
+    }
+}
+
```

Comment:
Let's make UIVIew(s) support BorderDrawable by default. 

I made it for CornerRoundDrawable as well. Only ShadowDrawable can't support it since it requires some variables to be retained. Maybe we can update ShadowDrawable so that if you don't have a shadow layer then it will use the view layer, I will think about it later.